comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Comment(TimestampMixin, models.Model):
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    recipe = models.ForeignKey('recipes.Recipe', on_delete=models.CASCADE)
+    text = models.TextField(max_length=2000)
+
+    class Meta:
+        ordering = ['-created_at']
+        verbose_name_plural = 'comments'
```

Comment:
we only need to set a custom `verbose_name_plural` when the plural word is different than just adding `s` to the model name, like `Story` -> `stories`

https://docs.djangoproject.com/en/2.2/ref/models/options/#django.db.models.Options.verbose_name_plural